### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by pavroo** [goastian/midori-desktop#34] [General]
Hi
I have 2 suggestions to deb packaging Midori

1. Add copyright file to:
/usr/share/doc/midori/

2. Add postinst file to DEBIAN directory:
`#!/bin/sh

set -e

if [ "$1" = "configure" ]; then
	update-alternatives --install /usr/bin/midori x-www-browser /usr/bin/midori 40
fi`

and prerm file to DEBIAN dir as well:

`#!/bin/sh

set -e

if [ "$1" = "remove" ]; then
	update-alternatives --remove x-www-browser /usr/bin/midori
fi`

**Comment by Ponchale** [comment]
checking.....
